src/pgrn-custom-scan.c

```diff
@@ -181,19 +183,59 @@ PGrnBeginCustomScan(CustomScanState *customScanState,
 					EState *estate,
 					int eflags)
 {
-	grn_obj *sourcesTable;
+	PGrnScanState *state = (PGrnScanState *) customScanState;
+	TupleDesc tupdesc =
+		RelationGetDescr(customScanState->ss.ss_currentRelation);
 	Relation index = PGrnChooseIndex(customScanState->ss.ss_currentRelation);
+	grn_obj *sourcesTable;
+
 	if (!index)
 		return;
 	sourcesTable = PGrnLookupSourcesTable(index, ERROR);
```

Comment:
```suggestion
	sourcesTable = PGrnLookupSourcesTable(index, ERROR);
	if (!sourceTable)
	  return;
```

Reply:
This is needless because we use `ERROR` for `PGrnLookupSourcesTable()`.